package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -125,6 +125,10 @@ local primary_addrs = {
 			'om2pv4',
 			'om2p-hsv4',
 		}},
+		{'ath79', 'generic', {
+			'plasmacloud,pa300',
+			'plasmacloud,pa300e',
+		}},
```

Comment:
I've just merged #2170 - As both device-trees define the label-mac-device, this is now obsolete.

Reply:
thanks for the info. I have now dropped it.